src/parameters/parameters_flow_solver.cpp

```diff
@@ -94,6 +96,17 @@ void FlowSolverParam::declare_parameters(dealii::ParameterHandler &prm)
                               " isothermal>.");
         }
         prm.leave_subsection();
+        
+        prm.enter_subsection("time_refinement_study");
+        {
+            prm.declare_entry("number_of_times_to_solve", "4",
+                              dealii::Patterns::Integer(1, dealii::Patterns::Integer::max_int_value),
+                              "Number of times to run the flow solver during a time refinement study.");
+            prm.declare_entry("refinement_ratio", "0.5",
+                              dealii::Patterns::Double(0, 1.0),
+                              "Ratio between a timestep size and the next in a time refinement study, 0<r<1.");
+        }
+        prm.leave_subsection();
```

Comment:
We'd probably want to move this into a file called `parameters_time_refinement_study.cpp` like we have for grid refinement and manufactured convergence studies since they are used by the test and not the flow solver, you can open an issue for it and address it later if you want -- we only want flow related parameters here

Reply:
I agree that that's a better way to organize things. I'll open an issue.